lib/utils.js

```diff
@@ -0,0 +1,42 @@
+const sdk = require('postman-collection');
+
+/**
+ * Method to merge more than one PostmanVariableList instances to get one unique list
+ * Provide array in Decreasing order of their scope
+ *
+ * @param {Object} variableLists - Array of PostmanVariableList to be combined together
+ * @returns {PostmabVariableList} - PostmanVariableList instance containing variables of all variableLists provided
+ */
+function resolveVariables (variableLists) {
+  let variables = {},
+    variableListArray = [],
+    variableList = new sdk.VariableList();
+
+  if (variableLists.global) {
```

Comment:
Let's not have globals. environments, collection etc. Lets assume we are getting a unified list directly